src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -527,9 +530,8 @@ const ComposeForm = props => {
       setSubject(draftInProgress?.subject ?? draft.subject);
       setMessageBody(draftInProgress?.body ?? draft.body);
 
-      if (draft?.attachments) {
-        setAttachments(draft.attachments);
-      }
+      // Note: Drafts cannot save attachments, so draft.attachments is always empty
```

Comment:
Is this accurate? When I tried saving a draft with attachments, I got a warning that drafts cannot save attachments so I'm not sure what the purpose of this code was